CalibMuon/DTCalibration/interface/DTT0BaseCorrection.h

```diff
@@ -4,6 +4,8 @@
 /** \class DTT0BaseCorrection
  *  Base class to define t0 corrections
  *
+ *  $Date: 2012/03/02 19:47:31 $
```

Comment:
hi @tobias-pook - please remove these, they are obsolete (a sign that the merge of these developments has not accounted for evolution in CMSSW in the meantime?)